docs/fundamentals/networking/http/httpclient-migrate-from-httpwebrequest.md

```diff
@@ -0,0 +1,136 @@
+
+Let's start with some examples:
+
+#### Simple GET Request Using <xref:System.Net.HttpWebRequest>
+
+Here's an example of how the code might look:
+
+```c#
+using System.Net;
+
+HttpWebRequest request = WebRequest.CreateHttp(uri);
+using WebResponse response = await request.GetResponseAsync();
+```
+
+#### Simple GET Request Using <xref:System.Net.Http.HttpClient>
+
+Here's an example of how the code might look:
+
+```c#
+using System.Net.Http;
+
+using HttpClient client = new();
+using HttpResponseMessage message = await client.GetAsync(uri);
+```
+
+#### Simple POST Request Using <xref:System.Net.HttpWebRequest>
+
+Here's an example of how the code might look:
+
+```c#
+using System.Net;
+
+HttpWebRequest request = WebRequest.CreateHttp(uri);
+request.Method = "POST";
+request.ContentType = "text/plain";
+await using Stream stream = await request.GetRequestStreamAsync();
+await stream.WriteAsync("Hello World!"u8.ToArray());
+using WebResponse response = await request.GetResponseAsync();
+Memory<byte> buffer = new byte[1024];
+await using Stream responseStream = response.GetResponseStream();
+```
+
+#### Simple POST Request Using <xref:System.Net.Http.HttpClient>
+
+Here's an example of how the code might look:
+
+```c#
+using System.Net.Http;
+
+using HttpClient client = new();
+using HttpResponseMessage responseMessage = await client.PostAsync(uri, new StringContent("Hello World!"));
+```
+
+## 1:1 API Mapping Table
+
+Developers should be aware that `ServicePointManager` is a static class, meaning that any changes made to its properties will have a global effect on all newly created `ServicePoint` objects within the application. For example, modifying a property like `ConnectionLimit` or `Expect100Continue` will impact every new ServicePoint instance.
+
+### <xref:System.Net.ServicePointManager> Properties Mapping
+
+| <xref:System.Net.ServicePointManager> Old API | New API | Notes |
+|---------|----------------------|-------|
+| `CheckCertificateRevocationList` | <xref:System.Net.Http.SocketsHttpHandler.SslOptions>.CertificateRevocationCheckMode | See TODO: for usage |
+| `DefaultConnectionLimit` | <xref:System.Net.Http.SocketsHttpHandler.MaxConnectionsPerServer> | TODO |
+| `DnsRefreshTimeout` | No equivalent API | Custom implementation needed |
+| `EnableDnsRoundRobin` | No equivalent API | Custom implementation needed |
+| `EncryptionPolicy` | No equivalent API | TODO |
+| `Expect100Continue` | <xref:System.Net.Http.Headers.HttpRequestHeaders.ExpectContinue> | TODO |
+| `MaxServicePointIdleTime` | <xref:System.Net.Http.SocketsHttpHandler.PooledConnectionIdleTimeout> | TODO |
+| `MaxServicePoints` | No equivalent API | TODO |
+| `ReusePort` | No direct equivalent API | Can be done over <xref:System.Net.Http.SocketsHttpHandler.ConnectCallback>. |
```

Comment:
"Can be done over ..." wording does not feel (stylistically) right, maybe just "See ConnectCallback" and add a paragraph with something like "Some ServicePointManager members don't have their equivalent on HttpClient, but connection customization can be performed on the underlying Socket instance by utilizing the ConnectCallback" before the tables?